brave/src/main/java/com/linecorp/armeria/client/brave/ClientRequestContextAdapter.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.brave;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.client.ClientRequestContext;
+import com.linecorp.armeria.common.RequestHeadersBuilder;
+import com.linecorp.armeria.common.RpcRequest;
+import com.linecorp.armeria.common.SerializationFormat;
+import com.linecorp.armeria.common.SessionProtocol;
+import com.linecorp.armeria.common.logging.RequestLog;
+import com.linecorp.armeria.common.logging.RequestLogAvailability;
+import com.linecorp.armeria.internal.brave.SpanContextUtil;
+import com.linecorp.armeria.internal.brave.SpanTags;
+
+import brave.http.HttpClientAdapter;
+
+/**
+ * Wraps a pair of {@link ClientRequestContext} and {@link RequestHeadersBuilder} in an {@link
+ * brave.http.HttpClientRequest}.
+ */
+final class ClientRequestContextAdapter {
+    /**
+     * @param ctx used to parse http properties
```

Comment:
Here, you need to insert some summary, like `Returns a new {@link brave.http.HttpClientRequest} from the given {@link ClientRequestContext} and {@link RequestHeadersBuilder}.

Reply:
that's studdering. I'll just delete it. thanks

Reply:
thanks for saving me.. the checkstyle being slightly under the type made me scratch head. I'm more used to things that align to the actual line of code.